### PR TITLE
Improve error handling, added testcoverage

### PR DESCRIPTION
**Pull request by staabm**
Before landing we should have some tests https://github.com/staabm/annotate-pull-request-from-checkstyle/issues/5

### CODE REVIEW

**Comment by staabm** [comment]
@localheinz maybe you have an idea why the github action based sanity tests in this PR dont work? it seems the action doesnt even start..

**Comment by localheinz** [comment]
@staabm 

Looks like the workflow definition was in the wrong directory - I have merged it with `.github/workflows/continuous-integration.yml`. 

I hope you don't mind me pushing into this branch!

**Comment by staabm** [comment]
Thank you so much.

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -16,24 +16,35 @@ jobs:
...
 
       - name: "Install PHP with extensions"
-        uses: "shivammathur/setup-php@1.8.1"
+        uses: "shivammathur/setup-php@v1"
         with:
           coverage: "none"
           extensions: "simplexml"
           php-version: "7.0"
 
-      - name: "Cache dependencies installed with composer"
```
Comment (staabm) [inline .github/workflows/continuous-integration.yml]: deleted this unnecessary complexity, because the build is wicked fast without caching

**Pull request merged**